PetFeeding/src/main/java/com/epam/ConnectionSource.java

```diff
@@ -0,0 +1,60 @@
+package com.epam;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class ConnectionSource {
+    // JDBC driver name and database URL
+    static final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/petsdb";
+
+    //  Database credentials
+    private static final String USER = "root";
+    private static final String PASS = "password";
```

Comment:
Don't hardcode passwords, use properties file that is not committed to git

Reply:
fixed